android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -14,22 +14,22 @@ public class Snapshot {
   public static final String HEIGHT = "height";
   public static final String ORIGIN_X = "originX";
   public static final String ORIGIN_Y = "originY";
-  public static final String GLOBAL_ORIGIN_X = "globalOriginX";
-  public static final String GLOBAL_ORIGIN_Y = "globalOriginY";
+  public static final String ABSOLUTE_ORIGIN_X = "globalOriginX";
```

Comment:
This rename introduces inconsistency between identifier and actual value, what's the purpose of doing so?

Reply:
@kmagiera don't like `GLOBAL` prefix - https://github.com/software-mansion/react-native-reanimated/pull/3827#discussion_r1054337375 but if we want to change the string `globalOriginX` this requires a lot of renaming in JS code (types and predefined animations) which is definitely out of the scope of this PR.

Reply:
maybe just revert these changes. I didn't realize before that these weren't introduced in this diff. If it's outside ot the scope of this change maybe lets keep it the way it was. I don't like these names but such long diff is not a good place to refactor such things anyways.